lasagne/nonlinearities.py

```diff
@@ -107,7 +107,6 @@ class ScaledTanH(object):
 
     By carefully matching :math:`\\alpha` and :math:`\\beta`, the nonlinearity
     can also be tuned to preserve the mean and variance of its input:
-
```

Comment:
You missed this one... it will break the rendering of the documentation. Please always check the complete diff!